articles/access_controls.md

```diff
@@ -1,58 +1,57 @@
 ---
-title: "Access and Content Restrictions"
+title: "Access Controls and Data Governance: Sharing Settings and Conditions for Use"
```

Comment:
Why would it be Access Controls and ...Sharing settings and Conditions for use isn't access controls sharing settings and conditions for use?

I would suggest "Data Governance: sharing settings and Conditions for use" but that might make it less obvious someone naive to know that it is the right header.